plugins/main/host-device/README.md

```diff
@@ -1,17 +1,25 @@
 # host-device
+
 Move an already-existing device into a container.
 
+## Overview
+
 This simple plugin will move the requested device from the host's network namespace
-to the container's. Nothing else will be done - no IPAM, no addresses.
+to the container's. IPAM configuration can be used for this plugin.
+
+## Network configuration reference
 
-The device can be specified with any one of three properties:
+The device can be specified with any one of four properties:
 * `device`: The device name, e.g. `eth0`, `can0`
 * `hwaddr`: A MAC address
 * `kernelpath`: The kernel device kobj, e.g. `/sys/devices/pci0000:00/0000:00:1f.6`
+* `deviceID`: A PCI address of network device, e.g `0000:00:1f.6`
```

Comment:
@s1061123  I didn't mark the deviceID as optional because it's similar to other existing options, such as hwaddr, device and kernelpath; at least one of these four options need to be provided.
Re the name of ``deviceID``, I give a second thought on it and maybe it should not be changed as that's the name used by device plugin [framework](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/deviceplugin/v1beta1/api.proto#L85) which in some cases might be referred with a different meaning other than PCI.

Reply:
In [above example](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/deviceplugin/v1beta1/api.proto#L77-L80), deviceID in k8s sometimes uses other than PCI address. So user may confuse if we name 'PCIBusID' as 'deviceID'. This plugin, `host-local` plugin is for host local device, not only for PCI also USB (i.e. USB network adapter) as well as other bus, so we should give appropriate name for that.

Reply:
Ok, fair enough, updated a new version with ``pciBusID``.